src/pathfinder.cpp

```diff
@@ -378,6 +381,32 @@ class Pathfinder {
 #endif
 };
 
+/** Helper class for the open list priority queue in the A* pathfinder
+ *  to sort the pathfinder nodes by cost.
+ */
+class PathfinderCompareHeuristic
+{
+	private:
+		Pathfinder* myPathfinder;
```

Comment:
`Pathfinder *myPathfinder;`
Please correct all the placement of pointers and references in this class definition. Feel free to omit redundant brackets or one-liner statements.